mkosi

```diff
@@ -2529,6 +2539,7 @@ def parse_args() -> CommandLineArguments:
     group.add_argument('-r', "--release", help='Distribution release to install')
     group.add_argument('-m', "--mirror", help='Distribution mirror to use')
     group.add_argument("--repositories", action=CommaDelimitedListAction, dest='repositories', help='Repositories to use', metavar='REPOS')
+    group.add_argument('--arch', help='Override the architecture of installation')
```

Comment:
let's call this "architecture" please, this is exotic enough to not benefit from abbreviation